api/src/server.ts

```diff
@@ -19,6 +20,8 @@ app.use("/workshop", routes.workshop);
 
 connectDB();
 
+app.use("/api", router); // connect routes (prefix as api), what is the lin 7 logic for?: import * as routes from "./routes/index"
```

Comment:
It imports modules user and workshop on [line 18 in this file](https://github.com/ChangePlusPlusVandy/PennsylvaniaWomenWork/blob/7c66c90a8230535c54a8b520ef6f40a4410add15/api/src/routes/index.ts#L18)